MISC/SetWindDirection.md

```diff
@@ -8,13 +8,18 @@ ns: MISC
 void SET_WIND_DIRECTION(float direction);
 ```
 
-```
-This is NOT a heading. It's a FLOAT value from 0.0-7.0. Look at this image:  
-i.imgur.com/FwVpGS6.png  
-----------------------------  
-This appears to be a heading, in radians (0-2*pi) -PNWParksFan  
-```
+Sets the wind direction.
 
 ## Parameters
-* **direction**: 
+* **direction**: the [wind direction](https://en.wikipedia.org/wiki/Wind_direction) in radians
 
+## Examples
+```lua
+-- 180 degrees, wind will blow from the south
+SetWindDirection(3.1415)
```

Comment:
Use `math.pi` instead for accuracy. Perhaps include an example with `math.rad(180)` too for completeness?